docker-compose.yaml

```diff
@@ -5,29 +5,25 @@ services:
       context: .
       dockerfile: Dockerfile
     environment:
-      - SERVERFULL_RUNTIME_HTTPSERVER_ADDRESS=:8081
-      - SERVERFULL_RUNTIME_CONNSTATE_REPORTINTERVAL=5s
-      - SERVERFULL_RUNTIME_CONNSTATE_HIJACKEDCOUNTER=http.server.connstate.hijacked
-      - SERVERFULL_RUNTIME_CONNSTATE_CLOSEDCOUNTER=http.server.connstate.closed
-      - SERVERFULL_RUNTIME_CONNSTATE_IDLEGAUGE=http.server.connstate.idle.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_IDLECOUNTER=http.server.connstate.idle
-      - SERVERFULL_RUNTIME_CONNSTATE_ACTIVEGAUGE=http.server.connstate.active.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_ACTIVECOUNTER=http.server.connstate.active
-      - SERVERFULL_RUNTIME_CONNSTATE_NEWGAUGE=http.server.connstate.new.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER=http.server.connstate.new
-      - SERVERFULL_RUNTIME_LOGGER_OUTPUT=STDOUT
-      - SERVERFULL_RUNTIME_LOGGER_LEVEL=INFO
-      - SERVERFULL_RUNTIME_STATS_OUTPUT=DATADOG
-      - SERVERFULL_RUNTIME_STATS_DATADOG_PACKETSIZE=32768
-      - SERVERFULL_RUNTIME_STATS_DATADOG_TAGS=
-      - SERVERFULL_RUNTIME_STATS_DATADOG_FLUSHINTERVAL=10s
-      - SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS=statsd:8125
-      - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
-      - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - NEXPOSE_HOST
-      - NEXPOSE_USERNAME
-      - NEXPOSE_PASSWORD
-      - NEXPOSE_PAGESIZE
+      SERVERFULL_RUNTIME_HTTPSERVER_ADDRESS: :8081
+      SERVERFULL_RUNTIME_CONNSTATE_REPORTINTERVAL: 5s
+      SERVERFULL_RUNTIME_CONNSTATE_HIJACKEDCOUNTER: http.server.connstate.hijacked
+      SERVERFULL_RUNTIME_CONNSTATE_CLOSEDCOUNTER: http.server.connstate.closed
+      SERVERFULL_RUNTIME_CONNSTATE_IDLEGAUGE: http.server.connstate.idle.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_IDLECOUNTER: http.server.connstate.idle
+      SERVERFULL_RUNTIME_CONNSTATE_ACTIVEGAUGE: http.server.connstate.active.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_ACTIVECOUNTER: http.server.connstate.active
+      SERVERFULL_RUNTIME_CONNSTATE_NEWGAUGE: http.server.connstate.new.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER: http.server.connstate.new
+      SERVERFULL_RUNTIME_LOGGER_OUTPUT: STDOUT
+      SERVERFULL_RUNTIME_LOGGER_LEVEL: INFO
+      SERVERFULL_RUNTIME_STATS_OUTPUT: NULL
+      SERVERFULL_RUNTIME_SIGNALS_INSTALLED: OS
+      SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS: 15 2
+      NEXPOSE_HOST: ""
```

Comment:
Task: these variable names are inaccurate. It'd be something like `VULNHYDRATOR_NEXPOSE_HOST`. I'd highly recommend documenting all the environment variables required in the README

Reply:
Suggestion: I'd highly recommend documenting all the environment variables required in the README

Reply:
Will do. Thanks!